build.sh

```diff
@@ -121,6 +121,8 @@ if hasArg -n; then
     INSTALL_TARGET=""
     LIBUCXX_BUILD_DIR=${LIB_BUILD_DIR}
 else
+    # TODO: should this be passed to cmake below instead of `LIB_BUILD_DIR`?
+    # shellcheck disable=SC2034
```

Comment:
`LIB_BUILD_DIR` gets passed to `cmake` on line 179 and this variable is unused -- I think we probably want to pass `LIBUCXX_BUILD_DIR` to `cmake` instead.

Reply:
I went to do this, but I think we probably don't want to "clean" the `LIBUCXX_BUILD_DIR` if it's the `lib` dir of a conda prefix, since that would nuke any other shared objects and potentially render the environment inoperable.

Reply:
nevermind, there are several breakages when instead using `LIBUCXX_BUILD_DIR` so I'm just going to remove that unused variable entirely.